tests/utils/validator_manager.go

```diff
@@ -905,15 +916,26 @@ func InitializeEndERC20Validation(
 	subnet interfaces.SubnetTestInfo,
 	stakingManager *erc20tokenstakingmanager.ERC20TokenStakingManager,
 	validationID ids.ID,
+	force bool,
```

Comment:
For a function this short, I have a slight preference for splitting it into two different functions to call the different underlying functions. This makes it more clear what the caller is doing instead of having a list of booleans passed into a function.

Reply:
Agreed.